recipes/protobuf/all/conanfile.py

```diff
@@ -58,23 +79,32 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+            if Version(self.version) >= "3.22.0" and is_msvc(self):
```

Comment:
```suggestion
            if Version(self.version) >= "3.22.0" and is_msvc(self):
                # Protobuf requires absl::abseil_dll
                # https://github.com/protocolbuffers/protobuf/blob/v25.3/cmake/abseil-cpp.cmake#L40
```

Reply:
@valgur please, include the comments in the recipe too, Protobuf is hard to track.

Reply:
Already did. :slightly_smiling_face:  Thanks for helping out with these.

Reply:
Thank you!